pycon/finaid/context_processors.py

```diff
@@ -9,6 +9,7 @@ def financial_aid(request):
         "show_finaid_status_button": has_application(request.user),
         "show_finaid_review_button": is_reviewer(request.user),
         "show_finaid_download_button": is_reviewer(request.user),
+        "show_finaid_receipt_form": has_application(request.user),
```

Comment:
Maybe add a `# FIXME - xxxx` comment here so we don't forget to update this with the real criteria for when to display the button.